pkg/crc/machine/driver.go

```diff
@@ -57,3 +57,22 @@ func setDiskSize(host *host.Host, diskSizeGiB int) error {
 
 	return updateDriverValue(host, diskSizeSetter)
 }
+
+func setSharedDirPassword(host *host.Host, password string) error {
+	driver, err := loadDriverConfig(host)
+	if err != nil {
+		return err
+	}
+
+	if len(driver.SharedDirs) == 0 {
+		return nil
+	}
+
+	sharedDir := []libmachine.SharedDir{}
+	for _, dir := range driver.SharedDirs {
+		dir.Password = password
```

Comment:
so we assume all shares share the same password.

Reply:
yes, for windows its based on the user account, so shares themselves don't have passwords i am not considering about sshfs here, when we implement that we might have to change this